README.md

```diff
@@ -0,0 +1,75 @@
+# IOXIO Dataspace example app
+
+This demo application is designed to show a practical example on how to create an
+application using Dataspace technologies.
+
+It consists of simple Python backend that is responsible for authentication and data
+retrieval and React-based frontend application.
+
+You can check the [online demo](https://example-app.demos.ioxio.dev) or run it locally
+(check the [configuration](#configuration) section for the instructions).
+
+Main idea is to demonstrate how to:
+
+- Perform authentication in a dataspace
+- Use the application credentials
+- Retrieve data products from Product Gateway
```

Comment:
I'd move the `Retrieve data products from Product Gateway` to be the first one, as it's the most important part in my opinion.